conf/routes

```diff
@@ -51,10 +51,17 @@ GET        /hyper/tags                    controllers.HyperMediaApi.tags
 GET        /hyper/tags/:id                controllers.HyperMediaApi.tag(id: Long)
 OPTIONS    /*all                          controllers.HyperMediaApi.preflight(all: String)
 
+# Tag export for inCopy
 GET        /tags/export/all               controllers.ReadOnlyApi.getTagsAsXml
 GET        /tags/export/single/:id        controllers.ReadOnlyApi.tagAsXml(id: Long)
 GET        /tags/export/merges/:since     controllers.ReadOnlyApi.mergesAsXml(since: Long)
 
+# More newspaper integration
+GET       /tools/newspaperintegration/tags/modified/:since          controllers.ReadOnlyApi.modifiedAsXml(since: Long)
```

Comment:
We are going to have to change these urls in the consumers, so if we want something less ugly :D